packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -109,6 +110,15 @@ class MediaQueryData {
   ///  * [MediaQuery.textScaleFactorOf], a convenience method which returns the
   ///    textScaleFactor defined for a [BuildContext].
   final double textScaleFactor;
+  
+  /// The current brightness mode of the host platform.
+  /// 
+  /// For example, starting in Android Pie, battery saver mode asks all apps to
+  /// render in a "dark mode". That change is reflected in this property.
+  /// 
+  /// Not all platforms necessarily support a concept of brightness mode. Those
+  /// platforms will report [Brightness.light] in this property.
+  final ui.Brightness platformBrightness;
```

Comment:
Is the `ui.` qualifier needed?

Reply:
I changed this file's import to bring in Brightness without a qualifier.